gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/policy/HiveSnapshotRegistrationPolicy.java

```diff
@@ -52,7 +52,7 @@
 
   protected final Optional<Pattern> snapshotPathPattern;
 
-  protected HiveSnapshotRegistrationPolicy(State props) throws IOException {
+  public HiveSnapshotRegistrationPolicy(State props) throws IOException {
```

Comment:
why this is needed?

Reply:
Thanks for caching, unintended change, will address